engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.io.File
+import java.net.URI
+
+class URIExt(uri: URI) {
+  def withFileSchemeDefault: URI = if (uri.isAbsolute) uri else new File(uri.getSchemeSpecificPart).toURI
```

Comment:
Can you add a test for that? What case will work other way after this change? Is it the case when `new URI(value)` throws an exception?

Reply:
Before:
Any URI in a config class would default to a file if no scheme were specified.
After:
The standard Ficus reader is used for a URI in a config class. If you want a default scheme, use the extension method.

Reply:
I have added a test.